Makefile

```diff
@@ -28,6 +28,7 @@ lint-client:
 	npm run lint:css --silent
 	npm run lint:js --silent
 	npm run lint:format --silent
+	npm run lint:project
```

Comment:
Maybe this section should just become `npm run lint`, since it's basically duplicating the definition in package.json (but apparently missing `npm run lint:ts`)?

Reply:
I might leave this one out of this PR, it's possible we intentionally want this split this way because of the way we want to use `--silent`.

I'll raise a separate PR with a clean up of these linting runners and get some feedback on that one in isolation.